common/src/client/rpc.rs

```diff
@@ -0,0 +1,206 @@
+use crate::pack::Pack;
+use anyhow::Result;
+use rand::rngs::OsRng;
+use solana_client::rpc_client::RpcClient;
+use solana_client::rpc_config::RpcSendTransactionConfig;
+use solana_sdk::commitment_config::CommitmentConfig;
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::signature::{Keypair, Signature, Signer};
+use solana_sdk::transaction::Transaction;
+use spl_token::instruction as token_instruction;
+use spl_token::pack::Pack as TokenPack;
+
+pub fn create_account_rent_exempt(
+    client: &RpcClient,
+    payer: &Keypair,
+    data_size: usize,
+    owner: &Pubkey,
+) -> Result<Keypair> {
+    let account = Keypair::generate(&mut OsRng);
+
+    let signers = [payer, &account];
+
+    let lamports = client.get_minimum_balance_for_rent_exemption(data_size)?;
+
+    let create_account_instr = solana_sdk::system_instruction::create_account(
+        &payer.pubkey(),
+        &account.pubkey(),
+        lamports,
+        data_size as u64,
+        owner,
+    );
+
+    let instructions = vec![create_account_instr];
```

Comment:
Could this be slice::from_ref

Reply:
Indeed it could, though I find that less clear without any obvious benefit. Admittedly I had to look up docs for that function. :P